release-notes/2024-07/README.md

```diff
@@ -0,0 +1,207 @@
+# SystemLink Enterprise release 2024-07 Release Notes
+
+The 2024-07 release bundle for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## New Features and Behavior changes
+
+- Align the x-axis of your [data table](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/visualizing-data-tables-in-a-data-space.html) plots to zero.
+- Decimate the data in your [Data Spaces](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/visualizing-data-tables-in-a-data-space.html) plots using Lossy, Max/Min, or Entry/Exit modes.
+- Filter and save a custom view of a [test plans](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/creating-a-test-plan.html) grid.
+- SystemLink Enterprise adds the [workspace](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/creating-a-workspace.html) property to products. For new products, assign a workspace so you can control access. For products already in your system, SystemLink Enterprise assigns a workspace using the following logic.
+
+| | |
+|--|--|
+| The product has test results. | SystemLink Enterprise assigns the workspace associated with the first test results listed for the product. |
+| The product does not have test results. | SystemLink Enterprise assigns the default workspace.|
+
+- `helium-serviceregistry 0.16.7`
+    - The Alarm Service privileges are exposed under the `Alarms` category in the security UI.
+- `testmonitorservice 0.24.22`
+    - Products have workspaces.
+    - This update includes a PostgreSQL migration to add workspaces to each product.
+    - Added new Helm chart values to facilitate the PostgreSQL migration.
+        - This update includes a new required whitelisted API key secret. This secret is used to retrieve workspace information during the migration.
```

Comment:
FYI @vpstoynova I've got [a PR up](https://dev.azure.com/ni/Users/_git/Documentation%20Reviews/pullrequest/758039) to include this in the ni.com docs as well.